docs/architecture/decisions/0004-create-an-oidc-oauth-server-for-get-an-identity.md

```diff
@@ -0,0 +1,53 @@
+# Create an oidc/oauth server to underpin get an identity
+
+27 July 2022
+
+## Status
+
+Accepted
+
+## Context
+
+Get an Identity service started life as "Teacher Identity" where it went through Discovery and Alpha phases. Although successful, it was clear that technically there many risks in creating an unmanageable highly complex web of software application components. We had identified and accepted (as a cross service intiative) a small set of common goals and objectives. The main ones being:
+
+* Be able to more easily identify our user records and information across our service line (10-12 discreet digital services).
+* Better inform policy making and service design by being able to better track a user across services.
+* Create better user experiences by reducing the amount of times we ask for core personal data.
+* More securely ask for, store, monitor, share and grant access to our users PII data.
+* Create better user experiences by being able to use (and link records to) other DfE data sets to gain insights into useage, workforce patterns etc. that allow us to save teachers time and effort in using our services.
+* Enable better interventions (by being able to more accurately and reliably identify a user securely) it allows us greater opportunities to automate services such as teacher training incentive payments, automate safeguarding measures etc.
+* More easily fix data gaps and errors.
+
+There are many ways to implement this kind of data integration and single sign on functionality. There are indeed a number of GOV solutions in this space. We have considered using:
+
+* [DfE sign](https://services.signin.education.gov.uk/). This service doesn't meet our use case because it is DfE Sign-in is how schools and other education organisations access DfE online services. Our users access our services as individuals outside of an educational organisation e.g. apply for a job, apply for initial training are not services where an organisation is relevant.
```

Comment:
```suggestion
* [DfE Sign-in](https://services.signin.education.gov.uk/). This service doesn't meet our use case because DfE Sign-in is how schools and other education organisations access DfE online services. Our users access our services as individuals outside of an educational organisation e.g. apply for a job, apply for initial training. These are not services where an organisation is relevant.
```